ILRepack.IntegrationTests/Scenarios/DotNet462Application/App.config

```diff
@@ -1,13 +1,13 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2" />
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>
```

Comment:
It makes no sense to change target .NET Framework version here. .NET Framework 4.6.2 is currently the minimal 4.x version to be supported by Microsoft.